bundles/org.openhab.binding.omnilink/README.md

```diff
@@ -147,6 +146,56 @@ The devices support some of the following trigger channels:
 | `switch_press_event`          | Event sent when an ALC, UPB, Radio RA, or Starlite switch is pressed.                | `dimmable`, `upb`                   |
 
 
+## Rule Actions
+
+This binding includes a rule action, which allows synchronizing the controller time to match the openHAB system time with a user specified zone.
+There is a separate instance for each contoller, which can be retrieved through:
+
+:::: tabs
```

Comment:
Curious: How can this be tested? GitHub doesn't display it correctly, so not sure how to verify.

Reply:
I think that the only way right now is testing by doing. I'm not sure that there is another good way to test it.